docs/source/getting-started/installation.rst

```diff
@@ -7,61 +7,122 @@ Installation
 
-.. code-block:: console
+.. tab-set::
 
-  $ # SkyPilot requires python >= 3.7. For Apple Silicon, use >= 3.8.
-  $ # Recommended: use a new conda env to avoid package conflicts.
-  $ conda create -y -n sky python=3.8
-  $ conda activate sky
+    .. tab-item:: Nightly (recommended)
 
-  $ # Choose an extra (default: [aws])
-  $ pip install skypilot
-  $ # pip install "skypilot[gcp]"
-  $ # pip install "skypilot[azure]"
-  $ # pip install "skypilot[kubernetes]"
-  $ # pip install "skypilot[lambda]"
-  $ # pip install "skypilot[ibm]"
-  $ # pip install "skypilot[scp]"
-  $ # pip install "skypilot[oci]"
-  $ # pip install "skypilot[all]"
+        .. code-block:: shell
 
-SkyPilot currently supports these cloud providers: AWS, GCP, Azure, IBM, OCI, SCP, Lambda Cloud, and Cloudflare (for R2 object store).
-If you only have access to certain clouds, use any combination of
-the pip extras above (e.g., :code:`"[aws,gcp]"`) to reduce the
-dependencies installed.
+          # SkyPilot requires python >= 3.7. For Apple Silicon, use >= 3.8.
+          # Recommended: use a new conda env to avoid package conflicts.
+          conda create -y -n sky python=3.8
+          conda activate sky
 
+          pip install skypilot-nightly
 
-To get the latest features/updates, either install the nightly build:
+          # Or, choose an extra (defaults to [aws]):
+          pip install "skypilot-nightly[aws]"
+          pip install "skypilot-nightly[gcp]"
+          pip install "skypilot-nightly[azure]"
+          pip install "skypilot-nightly[kubernetes]"
+          pip install "skypilot-nightly[lambda]"
+          pip install "skypilot-nightly[ibm]"
+          pip install "skypilot-nightly[scp]"
+          pip install "skypilot-nightly[oci]"
+          pip install "skypilot-nightly[all]"
 
-.. code-block:: console
+    .. tab-item:: Latest Release
+
+        .. code-block:: shell
+
+          # SkyPilot requires python >= 3.7. For Apple Silicon, use >= 3.8.
+          # Recommended: use a new conda env to avoid package conflicts.
+          conda create -y -n sky python=3.8
+          conda activate sky
+
+          pip install skypilot
+
+          # Or, choose an extra (defaults to [aws]):
+          pip install "skypilot[aws]"
+          pip install "skypilot[gcp]"
+          pip install "skypilot[azure]"
+          pip install "skypilot[kubernetes]"
+          pip install "skypilot[lambda]"
+          pip install "skypilot[ibm]"
+          pip install "skypilot[scp]"
+          pip install "skypilot[oci]"
+          pip install "skypilot[all]"
+
+To use a subset of the clouds, you can combine some of the pip extras above to
+reduce the dependencies installed:
 
-  $ pip install -U "skypilot-nightly[all]"
+.. code-block:: shell
 
-or install from source:
+  # Nightly:
+  pip install -U "skypilot-nightly[aws,gcp]"
+  # Latest release:
+  pip install -U "skypilot[aws,gcp]"
+
+To get the latest features, either use a nightly build (the **Nightly**
```

Comment:
Nit: Wondering should we have the source in the tabs above as well?

Reply:
Good call, done.

Reply:
Oh, I was wondering if "install from source" can be another tab as `From Source` in the installation instruction above?

Reply:
Done.